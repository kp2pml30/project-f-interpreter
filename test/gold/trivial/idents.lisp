(setq x 0)
(print null true false +11.1 -12.3 +30 -30)
