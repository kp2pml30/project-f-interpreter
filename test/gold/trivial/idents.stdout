Error: not a function setq; stack:
	{"overallPos":0,"line":1,"offset":0}
	{"overallPos":0,"line":1,"offset":0}